pkg/labels/labels.go

```diff
@@ -149,7 +149,7 @@ type Label struct {
 	// Source can be one of the values present in const.go (e.g.: LabelSourceContainer)
 	Source string `json:"source"`
 	// Mark element to be used to find unused labels in lists
-	deletionMark bool `json:"-"`
+	deletionMark bool
```

Comment:
Should this be put back? Do we not use this ?

Reply:
These changes are good as is for now because we are supposedly not importing this structure, but if we decide to do this in the future, we will have to revert this bit.

Reply:
Per the `encoding/json` docs: https://golang.org/pkg/encoding/json/#Marshal

> Struct values encode as JSON objects. Each exported struct field becomes a member of the object, using the field name as the object key, unless the field is omitted for one of the reasons given below.

Since this field is unexported, it will never be marshaled to JSON; thus the struct tag is ineffective.

```
[zsh|matt@nerr-2]:~/src/github.com/cilium/cilium/pkg/labels 0 (master) ± go vet .
# github.com/cilium/cilium/pkg/labels
./labels.go:152: struct field deletionMark has json tag but is not exported
```